pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test: TestSQLStatsRegions failed

pkg/ccl/testccl/sqlstatsccl/sqlstatsccl_test.TestSQLStatsRegions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/11057977?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/11057977?buildTab=artifacts#/) on release-23.1 @ [7ef1c09f591c03a9b8edbc0fefb63a29d278f5db](https://github.com/cockroachdb/cockroach/commits/7ef1c09f591c03a9b8edbc0fefb63a29d278f5db):


```
=== RUN   TestSQLStatsRegions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1ffa8747a1250ee574ba4234d629ddbd/logTestSQLStatsRegions367300578
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestSQLStatsRegions
    sql_stats_test.go:243: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1ffa8747a1250ee574ba4234d629ddbd/logTestSQLStatsRegions367300578
--- FAIL: TestSQLStatsRegions (199.73s)
=== RUN   TestSQLStatsRegions/secondary_tenant
    sql_stats_test.go:165: condition failed to evaluate within 3m0s: rows are not replicated to all regions [gcp-us-central1 gcp-us-west1]
    --- FAIL: TestSQLStatsRegions/secondary_tenant (180.93s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cluster-observability
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSQLStatsRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-30117

--- a/pkg/ccl/testccl/sqlstatsccl/BUILD.bazel
+++ b/pkg/ccl/testccl/sqlstatsccl/BUILD.bazel
@@ -2,6 +2,8 @@ load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 go_test(
     name = "sqlstatsccl_test",
+    # Explicitly indicate timeout as some of the tests (ie TestSQLStatsRegions) might take up to 3 min to run.
+    timeout = "moderate",
     srcs = [
         "main_test.go",
         "sql_stats_test.go",

--- a/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go
+++ b/pkg/ccl/testccl/sqlstatsccl/sql_stats_test.go
@@ -106,8 +106,8 @@ func TestSQLStatsRegions(t *testing.T) {
 	// change, the test sometimes is flakey because the latency budget allocated
 	// to closed timestamp propagation proves to be insufficient. This value is
 	// very cautious, and makes this already slow test even slower.
-	tdb.Exec(t, "SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval = '50 ms'")
-	tdb.Exec(t, `SET CLUSTER SETTING kv.closed_timestamp.lead_for_global_reads_override = '1500ms'`)
+	tdb.Exec(t, "SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval = '10ms'")
+	tdb.Exec(t, `SET CLUSTER SETTING kv.closed_timestamp.lead_for_global_reads_override = '500ms'`)
 	tdb.Exec(t, `ALTER TENANT ALL SET CLUSTER SETTING spanconfig.reconciliation_job.checkpoint_interval = '500ms'`)
 
 	tdb.Exec(t, "ALTER TENANT ALL SET CLUSTER SETTING sql.zone_configs.allow_for_secondary_tenant.enabled = true")

